styles/module/banner.css

```diff
@@ -0,0 +1,35 @@
+.banner {
+	background:url('../../img/banner.jpg') no-repeat;
+	background-size: cover;
+	display: flex;
+	align-items: flex-end;
+	height: 300px;
+}
+.banner ul {
+	display: none;
+}
+
+.next-line {clear: both;}
```

Comment:
Refactor selection to use nth-child selectors, so it's easier to manipulate for different resolutions.